subsys/bluetooth/services/lbs.c

```diff
@@ -90,7 +90,8 @@ BT_GATT_PRIMARY_SERVICE(BT_UUID_LBS),
 			       BT_GATT_CHRC_READ | BT_GATT_CHRC_NOTIFY,
 			       BT_GATT_PERM_READ, NULL, NULL, NULL),
 #endif
-	BT_GATT_CCC(lbslc_ccc_cfg_changed),
+	BT_GATT_CCC(lbslc_ccc_cfg_changed,
+		    BT_GATT_PERM_READ | BT_GATT_PERM_WRITE),
```

Comment:
HID was modified to allow configuration of the security level here. Maybe other services should do the same.

Reply:
This is not blocking - we don't use them so I don't care :)

Reply:
I see your point (checked in HID and handles permissions it a pretty neat way), but we're in an upmerge PR after all. I guess anyone interested can follow up after this PR.

Reply:
You are right. Let's ping @grochu .

Reply:
Looks good, no encryption or MiTM protection is needed to access this CCCD.